### PR TITLE
Add oauth-proxy-jxui-backend

### PR DESCRIPTION
**Pull request by macox**

### CODE REVIEW

File: env/parameters.tmpl.schema.json [hunk 1]
```diff
@@ -96,6 +96,18 @@
...
+      "title": "Dashboard auth id"
+    },
+    "dashboardAuthSecret": {
+      "type": "string",
+      "title": "Dashboard auth secret"
+    },
+    "dashboardAuthOrg": {
+      "type": "string",
+      "title": "Dashboard auth org"
+    },
```
Comment (cagiti) [inline env/parameters.tmpl.schema.json:110]: actually think these can go

Comment (macox) [reply env/parameters.tmpl.schema.json:110]: makes sense

File: env/parameters.yaml [hunk 2]
```diff
@@ -1,6 +1,9 @@
 adminUser:
   password: vault:arcalos-staging-mgmt/adminUser:password
   username: admin
+dashboardAuthID: vault:arcalos-staging-mgmt/dashboard/auth:id
+dashboardAuthOrg: arcalos-dashboard
```
Comment (cagiti) [inline env/parameters.yaml:5]: this can definitely go

File: env/parameters.tmpl.schema.json [hunk 3]
```diff
@@ -96,6 +96,18 @@
...
     },
+    "dashboardAuthID": {
+      "type": "string",
+      "title": "Dashboard auth id"
+    },
+    "dashboardAuthSecret": {
+      "type": "string",
+      "title": "Dashboard auth secret"
+    },
+    "dashboardAuthOrg": {
```
Comment (cagiti) [inline env/parameters.tmpl.schema.json:107]: this can go too

File: env/oauth-proxy-jxui-backend/values.tmpl.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+config:
+  clientID: "{{ .Parameters.dashboardAuthID }}"
+  clientSecret: "{{ .Parameters.dashboardAuthSecret }}"
+extraArgs:
+  prompt: select_account
+  cookie-secret: vault:arcalos-staging-mgmt/dashboard/cookie:secret
+  cookie-name: jxui-oidc
+  cookie-domain: ".{{ $rootDomain }}"
+  email-domain: "*"
+  oidc-issuer-url: https://id.cloudbees.com/
```
Comment (cagiti) [inline env/oauth-proxy-jxui-backend/values.tmpl.yaml:12]: this should be id.beescloud.com

**Pull request merged**